copier/vcs.py

```diff
@@ -46,6 +46,7 @@ def is_in_git_repo(path: StrOrPath) -> bool:
 
 def is_git_bundle(path: Path) -> bool:
     """Indicate if a path is a valid git bundle."""
+    path = path.resolve()
     with TemporaryDirectory(prefix=f"{__name__}.is_git_bundle.") as dirname:
         with local.cwd(dirname):
             git("init")
```

Comment:
You can also do:

```suggestion
    with TemporaryDirectory(prefix=f"{__name__}.is_git_bundle.") as dirname:
        git("-C", dirname, "init")
        return bool(git["-C", dirname, "bundle", "verify", path] & TF)
```

Both solutions are good, so do as you prefer 😄

Reply:
Oh well, I think I prefer yours: the least directory changes, the better. And that removes the exception on Windows :+1: